moderator/moderate/views.py

```diff
@@ -130,7 +129,10 @@ def show_event(request, e_slug, q_id=None):
     questions_q = Question.objects.filter(event=event).annotate(
         vote_count=Count("votes")
     )
-    questions = questions_q.order_by("-vote_count")
+    if user.is_superuser:
```

Comment:
Should this be `user.userprofile.is_admin` instead?

Reply:
Actually should be this be `event.archived or user.userprofile.is_admin` to reflect the logic expressed in the issue?
> When an events gets archived, the questions are ordered based on the vote count